core/src/main/java/io/grpc/internal/AbstractServerStream.java

```diff
@@ -228,17 +230,8 @@ public final void abortStream(Status status, boolean notifyClient) {
     // TODO(louiscryan): Investigate whether we can remove the notification to the client
     // and rely on a transport layer stream reset instead.
     Preconditions.checkArgument(!status.isOk(), "status must not be OK");
+    inboundPhase(Phase.STATUS);
     closeListener(status);
-    if (notifyClient) {
```

Comment:
When notifyClient == true, then the stream is supposed to still be active. Should we send RST_STREAM in that case?